mubench.pipeline/data/runner_interface.py

```diff
@@ -37,11 +32,9 @@ def __init__(self, jar_path: str, java_options: List[str]):
     @staticmethod
     def get(requested_version: StrictVersion, jar_path: str, java_options: List[str]) -> 'RunnerInterface':
         interfaces = RunnerInterface._get_interfaces()
-        matching_interfaces = [i for i in interfaces if i.version() == requested_version]
-        if matching_interfaces:
-            return matching_interfaces[0](jar_path, java_options)
-        else:
-            raise NoCompatibleRunnerInterface(requested_version)
```

Comment:
This error case doesn't completely go away with the new implementation, does it? Someone could specify a version that is smaller than any supported interface. Is there another case where wouldn't have a match? If this is the only case, we could maybe issue a warning and try the most recent interface?